homeassistant/components/fibaro.py

```diff
@@ -210,12 +211,20 @@ def setup(hass, config):
                          config[DOMAIN][CONF_PASSWORD],
                          config[DOMAIN][CONF_URL],
                          config[DOMAIN][CONF_PLUGINS])
+
+    def stop_fibaro(event):
+        """Stop Fibaro Thread."""
+        _LOGGER.info("Shutting down Fibaro Hub")
```

Comment:
I think this message is slightly excessive since it is basically just a progress log.

Reply:
I agree, but I just followed exactly what the Xiaomi and the Vera components are doing and saying.

Reply:
I see. If you agree, just remove it – you should be able to find examples of that as well :-)